### DISCUSSION TITLE
How to configure Renovate to auto-merge our own dependencies

### DISCUSSION THREAD

**Discussion by joedayz** [renovatebot/renovate#16877] [Request Help]
### What would you like Renovate to be able to do?

Hi guys, I am figuring out how to configure renovate to auto-merge my own dependencies.

It saves us some time when we release patches to our type libs.

Some advice about this?

Jose

### If you have any ideas on how this should be implemented, please tell us here.

I need some advice on starting, the feature exists or should work with it feature?

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by rarkins** [comment]
Search docs.renovatebot.com for “automerge”

**Comment by joedayz** [comment]
Hi, @rarkins:

I read the documentation, but I don't know if the "automerge" is for me.

Then, I have it information:

1. https://github.com/company/Quadim-SpringServiceBaseLib/

This project uses a dependency located in another project

` <dependency>
            <groupId>ai.quadim.api</groupId>
            <artifactId>quadim-typelib</artifactId>
            <version>${quadim-typelib-version}</version>
        </dependency>
`

2. https://github.com/company/Quadim-Typelib/blob/master/pom.xml

3. Both projects are published in this repo

`    <distributionManagement>
        <repository>
            <id>quadim</id>
            <name>Quadim Release Repository</name>
            <url>https://mvnrepo.cantara.no/content/repositories/releases/</url>
        </repository>
        <snapshotRepository>
            <id>cantara</id>
            <name>Quadim Snapshot Repository</name>
            <url>https://mvnrepo.cantara.no/content/repositories/snapshots/</url>
        </snapshotRepository>

    </distributionManagement>

    <repositories>
        
        <repository>
            <id>cantara-releases</id>
            <name>Cantara Release Repository</name>
            <url>https://mvnrepo.cantara.no/content/repositories/releases/</url>
        </repository>
        <repository>
            <id>cantara-snapshots</id>
            <name>Cantara Snapshot Repository</name>
            <url>https://mvnrepo.cantara.no/content/repositories/snapshots/</url>
        </repository>

        <repository>
            <id>codeguru-profiler</id>
            <name>codeguru-profiler</name>
            <url>https://d1osg35nybn3tt.cloudfront.net</url>
        </repository>

    </repositories>

`


Then, is "automerge" the solution for my case?


Please, let me know if it should be the solution for my case.


Jose

**Reply by rarkins** [reply]
Renovate's automerge feature is the only way you can automatically merge dependencies using Renovate, so it's either that feature or you need to use some other solution for merging
